resteasy-spring-feature-pack/common/src/main/resources/layers/standalone/resteasy-spring/layer-spec.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" ?>
+<layer-spec xmlns="urn:jboss:galleon:layer-spec:2.0" name="resteasy-spring">
+    <props>
+        <prop name="org.wildfly.rule.xml-path" value="/WEB-INF/web.xml,/web-app/listener/listener-class,org.jboss.resteasy.plugins.server.servlet.ResteasyBootstrap"/>
+    </props>
+    <packages>
```

Comment:
@jamezp @jfdenise With my understanding, the package here in the layer spec will load the modules inside the WildFly, am I correct?

Reply:
@jfdenise Okay now I understand. Thanks for the answer! :D